src/atoms/errorField/ErrorField.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+ 
+const ErrorField = ({errorText}) => {
+   return (
+       <h1>{errorText}</h1>
+   );
+};
+
+ErrorField.propTypes = {
+    errorText: PropTypes.string
+  };
+  
+ErrorField.defaultProps = {
+    errorText:''
```

Comment:
So now if we have used default props, then we can make use of it in appropriate way like : No Data Found... otherwise if it still comes as undefined we do not show anything which is similar to showing empty string.